### PR TITLE
Zubax GNSS firmware update manual

### PR DESCRIPTION
**Pull request by pavel-kirienko**

### CODE REVIEW

File: book/uavcan-node-firmware.md [hunk 1]
```diff
@@ -69,6 +69,11 @@ make px4esc.image
 ```
 The firmware image will be located at `firmware/build/org.pixhawk.px4esc-bldc-v1-1.0.<xxxxxxxx>.bin`, where `<xxxxxxxx>` is an arbitrary sequence of numbers and letters.
 
+## Zubax GNSS
+
+Please refer to the project page at <https://github.com/Zubax/zubax_gnss> to learn how to build and flash the firmware.
```
Comment (LorenzMeier) [inline book/uavcan-node-firmware.md]: The way to put links into .md files is like this:

```md
[Zubas GNSS documentation](https://github.com/Zubax/zubax_gnss)
```

Comment (pavel-kirienko) [inline book/uavcan-node-firmware.md]: I know. The intent was to make the link explicit. Do you still want me to change it?

Comment (LorenzMeier) [inline book/uavcan-node-firmware.md]: I would prefer to keep the style consistent. You can still refer to "Github repository Readme" or something if you like to make this more visible.

Comment (pavel-kirienko) [inline book/uavcan-node-firmware.md]: Done.

**Pull request merged**